.github/actions/code-scanning/action.yaml

```diff
@@ -0,0 +1,41 @@
+name: Code Scanning
+description: Common steps to build a Maven artifact
+inputs:
+  checkoutCode:
+    description: Wherever or not to checkout the code for the repo
+    required: true
```

Comment:
Would this ever be false?

Reply:
It can be false if you are checking out the code outside of the step, so within the `reusable-code-scanning.yml` action

Reply:
Yeah that's right, I don't think we have a cron on the individual repos but on merge, so will update their pipelines to use this reusable one. But I've changed the nightly checks pipeline in this repo to run nightly, so we're covered on both